static/src/javascripts/projects/common/modules/preferences/main.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'react',
```

Comment:
It might be a good idea to give the Closure Compiler a whirl, think Ophan uses it successfully

tbh, all out libs summed probably is bigger that jQuery, but microlibs give you more benefits than size, e.g. right (programmer friendly) lib for the job, ability to switch out one lib for another, makes upgrading easier